packages/core/src/crypto/webcrypto/CryptoKey.ts

```diff
@@ -0,0 +1,16 @@
+import type { KeyAlgorithm, KeyType, KeyUsage } from './types'
+import type { Key } from '../Key'
+
+import * as core from 'webcrypto-core'
+
+export class CryptoKey extends core.CryptoKey {
```

Comment:
I wanted to avoid the Credo prefix, but not really for any major specific reason. Can add it or keep it at `WebCryptoKey` as well. If we prefix it here with `Credo` I would like to do it for all the classes.